cmd/influx/v1_dbrp.go

```diff
@@ -88,28 +88,16 @@ func v1DBRPFindF(cmd *cobra.Command, args []string) error {
 	}
 	filter := influxdb.DBRPMappingFilterV2{OrgID: &orgID}
 
-	if v1DBRPFindFlags.ID != "" {
-		fID, err := influxdb.IDFromString(v1DBRPFindFlags.ID)
-		if err != nil {
-			return err
-		}
-		filter.ID = fID
+	if v1DBRPFindFlags.ID.Valid() {
```

Comment:
Should we return an `err` as an `else` case for these checks?

Reply:
Not needed, because I added them as IDVar parameters, they will be validated at argument parsing time.